src/data/Endpoint.js

```diff
@@ -46,7 +46,7 @@ const endpoints = {
     DEPOTS: `${laravelUrlPrefix}/${version}/customer/depot`,
     CUSTOMER_SETTINGS_SHOW: `${laravelUrlPrefix}/${version}/customer/setting/first`,
     TRANSACTION_GROUP_SETTING: `${laravelUrlPrefix}/${version}/transaction-groups/{1}`,
-    IDENTITIES: `${laravelUrlPrefix}/${version}/identities`,
+    IDENTITIES: `${laravelUrlPrefix}/${version}/identities/{0}`,
```

Comment:
Technically this is not a rest integration. We have getIdentities and getIdentity: http://api-test.carpal.me/v3/documentation ?

Also, now we will have to retest existing integrations of this function. Have we done that?

@pncsantos @julianideamotive

Reply:
@maartenhemmes why it isn't REST? We have main endpoint `identities` which should get all identities and we also have an option to pass identityId to get one identity, all previous functionalities will work the same way. I think this fulfills the REST resource naming guidelines.

The other thing is that we don't use identities collection (as whole collection) in app anyway, it was used only for finding this one identity using identityid and now we can do this by passing this id.

Reply:
@julianideamotive can we revert the existing api and just add a separate endpoint for fetching user identity instead. In this way we can focus on testing for the new endpoint rather than testing if there is a breaking changes being introduce.

Reply:
@pncsantos @maartenhemmes of course I can do that, but the only change here is the conditional param, so if we don't have it, it will work the same as previously. And as I mentioned, we don't use endpoint without param anywhere in app so nothing will break

anyway, I'll revert the change and create separate endpoint

Reply:
@pncsantos done

Reply:
@maartenhemmes looks okay now.